types/src/model/mod.rs

```diff
@@ -65,3 +69,33 @@ pub use self::{
         UnloadWallet, WalletCreateFundedPsbt, WalletDisplayAddress, WalletProcessPsbt,
     },
 };
+
+/// Models the data returned by Core for a script pubkey.
```

Comment:
I believe 'script pubkey' is definitely wrong but I don't know if it should be scriptPubkey or scriptPubKey. See #371

Reply:
I have gone with scriptPubKey to match Core.